manifest.json

```diff
@@ -6,7 +6,7 @@
   "version": "1.0.1",
   "author": "Dara Javaherian",
     
-  "permissions": ["tabs", "http://*/*", "https://*/*"],
+  "permissions": ["tabs", "*://*.youtube.com/*"],
```

Comment:
@darajava I don't believe the youtu.be domain needs to be included, since the extension only loads on youtube.com (`content_scripts` section in this file). I believe the syntax I used would match:

- `http://youtube.com`
- `https://youtube.com`
- `http://www.youtube.com`
- `https://www.youtube.com`
- any other subdomain of `youtube.com`